packages/tools/src/apm.coffee.md

```diff
@@ -1,7 +1,7 @@
 
 # `nikita.tools.apm`
 
-Install Atom packages with APM.
+Install actions Atom packages with APM.
```

Comment:
Action is not plural, the is one action which itself is composed of other actions

Reply:
OK